components/infobox/commons/infobox_person.lua

```diff
@@ -214,6 +217,32 @@ function Person:_setLpdbData(args, links, status, personType, earnings)
 	mw.ext.LiquipediaDB.lpdb_player(storageType .. '_' .. (args.id or self.name), lpdbData)
 end
 
+-- Allows this function to be used in /Custom
+function Person:getStandardNationalityValue(nationality)
+	if String.isEmpty(nationality) then
+		return nil
+	end
+
+	local nationalityToStore = Flags.CountryName(nationality)
+
+	if String.isEmpty(nationalityToStore) then
+		table.insert(
+			_warnings,
+			'"' .. nationality .. '" is not supported as a value for nationalities'
+		)
+	end
+
+	return nationalityToStore
```

Comment:
If input nationality is the empty string, then the function returns nil.
If input nationality is an incorrect string, then the function returns the empty string.
Is this intended?